documentation/src/main/asciidoc/public/getting-started/standalone/index.adoc

```diff
@@ -21,8 +21,8 @@
 include::../../components/_incubating-warning.adoc[]
 
 This guide will walk you through the initial steps required
-to index and query your entities using Hibernate Search,
-when your entities are not defined in Hibernate ORM.
+to index and query your entities using http://hibernate.org/search/documentation[Hibernate Search],
+when your entities are not defined in http://hibernate.org/orm/documentation[Hibernate ORM].
```

Comment:
Maybe stress the "not", because the link on "Hibernate ORM" will make it more prominent.

```suggestion
when your entities are **not** defined in http://hibernate.org/orm/[Hibernate ORM].
```